spec/openapi3.yaml

```diff
@@ -2085,7 +2085,7 @@ components:
       properties:
         mosaicNonce:
           type: integer
-          format: int32
+          format: int64
```

Comment:
Thanks for the int65, have a look just above, the required: tag. Should it be mosaicNonce instead of nonce?

Reply:
Well seen, fixed this as well!